apps/crn-server/src/utils/fetch-orcid.ts

```diff
@@ -1,6 +1,8 @@
+import { OrcidWork } from '@asap-hub/model';
 import Got from 'got';
 import get from 'lodash.get';
-import { CMSOrcidWork } from '../data-providers/user.data-provider';
+
+export type CMSOrcidWork = OrcidWork;
```

Comment:
This doesn't seem to be `import`ed anywhere else. Is there a reason to `export` it?